tests/test_utils.py

```diff
@@ -24,6 +25,9 @@
 )
 from stable_baselines3.common.vec_env import DummyVecEnv, SubprocVecEnv
 
+# a hack to get atari environment registered for 1.0.0 alpha 1
```

Comment:
why is it a hack? what is the recommended way that is not one?

Reply:
Gymnasium v1.0 removes implicit module loading which was used to register external environments, e.g., atari. 
We have updated ale-py but not cut a release, hoping to happen soon.
When alpha 2 is released, this will be fixed and the "hack" can be removed

Reply:
I have updated to alpha 2 and removed this code